emptytheme/functions.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+if ( ! function_exists( 'emptytheme_support' ) ) :
+	function emptytheme_support()  {
+
+		// Alignwide and alignfull classes in the block editor
+		add_theme_support( 'align-wide' );
+
+		// Adding support for core block visual styles.
+		add_theme_support( 'wp-block-styles' );
+
+		// Adding support for responsive embedded content.
+		add_theme_support( 'responsive-embeds' );
+
+		// Add support for experimental link color control.
+		add_theme_support( 'experimental-link-color' );
```

Comment:
I guess we should enable link color using the theme.json. global.settings.color.link = true;,

Reply:
Hi @kjellr, did you faced any problem enabling link color using theme.json global.settings.color.link = true;?

Reply:
Good call! I just added this in c75159f and it works fine. 👍